traefik/Changelog.md

```diff
@@ -1,5 +1,84 @@
 # Change Log
 
+## 35.1.0  ![AppVersion: v3.3.6](https://img.shields.io/static/v1?label=AppVersion&message=v3.3.6&color=success&logo=) ![Kubernetes: >=1.22.0-0](https://img.shields.io/static/v1?label=Kubernetes&message=%3E%3D1.22.0-0&color=informational&logo=kubernetes) ![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
+
+**Release date:** 2025-04-25
+
+* feat: ✨ versionOverride
+* feat(Traefik Hub): namespaces
+* feat(CRDs): remove APIAccess resource
+* chore(release): :rocket: publish v35.1.0 and CRDs v1.7.0
+
+**Upgrade Notes**
+
+Traefik v3.3.6 comes with a change request path, please read https://doc.traefik.io/traefik/v3.3/migration/v3/#request-path-sanitization.
```

Comment:
Wasn't it for v35.0.1 ?

For this v35.1.0, I guess we should give instructions on how to delete manually APIAccess ?